.github/workflows/README.md

```diff
@@ -0,0 +1,114 @@
+(ARO)](https://docs.microsoft.com/en-us/azure/openshift/intro-openshift) cluster
+so that the `Inspektor Gadget CI` workflow will run the integration tests on it.
+
+These are the steps to create the cluster using the CLI:
+
+```bash
+$ export SUBSCRIPTION=<mySubscription>
+$ export RESOURCEGROUP=<myResourceName>
+$ export CLUSTER=<myCluster>
+$ export VNET=<myVNET>
+$ export MASTSUB=<myMASTSUB>
+$ export WORKSUB=<myWORKSUB>
+
+# Set subscription so that we don't need to specify it at every command
+$ az account set --subscription $SUBSCRIPTION
+
+# Register resource providers
+$ az provider register -n Microsoft.RedHatOpenShift --wait
+$ az provider register -n Microsoft.Compute --wait
+$ az provider register -n Microsoft.Storage --wait
+$ az provider register -n Microsoft.Authorization --wait
+
+# Create resource group
+$ az group create --name $RESOURCEGROUP --location $LOCATION
+
+# Create virtual network and two empty subnets for the master and the worker nodes.
+$ az network vnet create --resource-group $RESOURCEGROUP --name $VNET --address-prefixes 10.0.0.0/22
+$ az network vnet subnet create --resource-group $RESOURCEGROUP --vnet-name $VNET --name $MASTSUB --address-prefixes 10.0.0.0/23 --service-endpoints Microsoft.ContainerRegistry
+$ az network vnet subnet create --resource-group $RESOURCEGROUP --vnet-name $VNET --name $WORKSUB --address-prefixes 10.0.2.0/23 --service-endpoints Microsoft.ContainerRegistry
+$ az network vnet subnet update --name master-subnet --resource-group $RESOURCEGROUP --vnet-name $VNET --disable-private-link-service-network-policies true
+
+# Create the cluster (Minimum 3 worker nodes must be used)
+$ az aro create --resource-group $RESOURCEGROUP --name $CLUSTER --vnet $VNET --master-subnet $MASTSUB --worker-count 3 --worker-subnet $WORKSUB
+```
+
+After executing the `az aro create` command, it normally takes about 35 minutes
+to create a cluster.
+
+Notice that creating an ARO cluster requires the `User Access Administrator`
+permissions. The following error is printed when your account does not have
+them:
+```bash
+$ az aro create --resource-group $RESOURCEGROUP --name $CLUSTER --vnet $VNET --master-subnet $MASTSUB --worker-count 3 --worker-subnet $WORKSUB
+# The client 'myemail@domain.com' with object id '<my-user-object-id>' does not have authorization to perform action 'Microsoft.Authorization/roleAssignments/write' over scope '/subscriptions/<mySubscription>/resourceGroups/<myResourceName>/providers/Microsoft.Network/virtualNetworks/<myVNET>/providers/Microsoft.Authorization/roleAssignments/a1b2c3d4-eeee-5555-ffff-6g7h8i9k0000' or the scope is invalid. If access was recently granted, please refresh your credentials
+```
+
+If we need to delete our cluster, it is enough to execute:
+```bash
+$ az aro delete --resource-group $RESOURCEGROUP --name $CLUSTER
+```
+
+Take into account that it will remove the entire resource group and all
+resources inside it.
+
```

Comment:
Took this from [documentation](https://docs.microsoft.com/en-us/azure/openshift/tutorial-delete-cluster#delete-the-cluster) (and it is the same [upstream](https://github.com/MicrosoftDocs/azure-docs/blob/main/articles/openshift/tutorial-delete-cluster.md#delete-the-cluster)) but it seems strange to me that removing the `aro` resource removes the entire resource group 🤔. I will make a test tomorrow to confirm it.

Reply:
My tests confirmed the documentation is not accurate. I updated this PR to use the instructions I used in my tests and I created a PR to improve the documentation: https://github.com/MicrosoftDocs/azure-docs/pull/90207